packages/hint/docs/user-guide/concepts/connectors.md

```diff
@@ -1,60 +1,65 @@
 # Connectors
 
-A `connector` is the interface between the `hint`s and the website
-you are testing. It is responsible for loading the website and exposing
-all the information to `webhint` such as resources, network data, etc.
+A `connector` is the interface between the `hint`s and the website you
+are testing. It is responsible for loading the website and exposing all
+the information to `hint` such as resources, network data, etc.
```

Comment:
Yeah, this is a good example of the name confusion you mentioned. I think "webhint" is better here as this applies beyond just the CLI. Perhaps if we drop the code-style formatting this will be less confusing because it won't look like a command, much like some similar changes you made elsewhere.
```suggestion
the information to webhint such as resources, network data, etc.
```